tsl/test/expected/continuous_aggs_errors.out

```diff
@@ -473,30 +473,37 @@ select time_bucket(100, timec), min(location), sum(temperature),sum(humidity)
 from conditions
 group by time_bucket(100, timec);
 ERROR:  parameter timescaledb.ignore_invalidation_older_than must be an integer for hypertables with integer time values
-\set ON_ERROR_STOP 1
 create materialized view mat_with_test( timec, minl, sumt , sumh)
 WITH ( timescaledb.continuous, timescaledb.refresh_lag = '2147483647')
 as
 select time_bucket(100, timec), min(location), sum(temperature),sum(humidity)
 from conditions
 group by time_bucket(100, timec);
--- Should print a useful error message, but not fail.
-CREATE MATERIALIZED VIEW IF NOT EXISTS mat_with_test
-WITH (timescaledb.continuous) AS
-SELECT time_bucket(100, timec), min(location), sum(temperature),sum(humidity)
-  FROM conditions
-GROUP BY time_bucket(100, timec);
-NOTICE:  continuous aggregate "mat_with_test" already exists, skipping
+ERROR:  invalid integer_now function
 \set ON_ERROR_STOP 0
-ALTER MATERIALIZED VIEW mat_with_test SET(timescaledb.refresh_lag = '1h');
-ERROR:  parameter timescaledb.refresh_lag must be an integer for hypertables with integer time values
-ALTER MATERIALIZED VIEW mat_with_test SET(timescaledb.refresh_lag = '2147483648');
-ERROR:  timescaledb.refresh_lag out of range
-ALTER TABLE conditions ALTER timec type bigint;
-ERROR:  cannot alter type of a column used by a view or rule
-\set ON_ERROR_STOP 1
-DROP MATERIALIZED VIEW mat_with_test;
-ALTER TABLE conditions ALTER timec type bigint;
+DROP TABLE conditions cascade;
+CREATE TABLE conditions (
+      timec       BIGINT       NOT NULL,
+      location    TEXT              NOT NULL,
+      temperature DOUBLE PRECISION  NULL,
+      humidity    DOUBLE PRECISION  NULL,
+      lowp        double precision NULL,
+      highp       double precision null,
+      allnull     double precision null
+    );
+select table_name from create_hypertable( 'conditions', 'timec', chunk_time_interval=> 100);
+ table_name 
+------------
```

Comment:
There is an `\set ON_ERROR_STOP 0` at the beginning here but there are no failing statements after. Is this intentional?